exporter/datadogexporter/utils/trace_helpers.go

```diff
@@ -24,7 +25,10 @@ import (
 // constants for tags
 const (
 	// maximum for tag string lengths
-	MaxTagLength = 200
+	MaxTagLength           = 200
+	ServiceLangOtel string = "otel"
+	// DefaultServiceName is the default name we assign a service if it's missing and we have no reasonable fallback
+	DefaultServiceName string = "unnamed-service"
```

Comment:
Do we do anything similar in the Trace Agent? We could link it here.

Reply:
updated, with comment that links to info from trace-agent